db/migrate/20250318202734_add_first_rss_published_at_to_episodes.rb

```diff
@@ -0,0 +1,9 @@
+class AddFirstRssPublishedAtToEpisodes < ActiveRecord::Migration[7.2]
+  def change
+    add_column :episodes, :first_rss_published_at, :timestamp
+
+    default_feeds = EpisodesFeed.where(feed_id: Feed.default)
+    episodes_in_default_feeds = Episode.published.where(id: default_feeds.pluck(:episode_id))
+    episodes_in_default_feeds.update_all("first_rss_published_at = published_at")
```

Comment:
It looks like right now, the SQL this is running is querying for 1000s of episode_ids, then running an update with them as params.  Which isn't the most efficient, when updating 100K rows.

To do this entirely in the db, I think you just need to change `.pluck` to a `.select`.  Then the whole thing runs in a single query.